builtin_set.go

```diff
@@ -364,18 +364,20 @@ func (so *setObject) estimateMemUsage(ctx *MemUsageContext) (estimate uint64, er
 		}
 		samplesVisited += 1
 
-		// We still want to account for both key and value if we return a non-zero value on error.
-		// This could otherwise skew the estimate when in reality key/value pairs contribute to
-		// mem usage together.
```

Comment:
I removed this on purpose to align with how we normally calculate mem usage, can refactor later if we notice this being an issue but I expect it to be rare.